### PR TITLE
bugfix: fix postgresql sql  script index name conflicts

### PR DESCRIPTION
**Pull request by gao1143632516**
In the same mode, in order to avoid the conflict of different table index names, it is best to spell the table name in the index name

- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did

fix the postgresql.sql index name conflicts

### CODE REVIEW

File: script/server/db/postgresql.sql
Comment (funky-eyes) [inline script/server/db/postgresql.sql:1]: Why do they conflict?

Comment (gao1143632516) [reply script/server/db/postgresql.sql:1]: CREATE INDEX idx_xid ON public.lock_table (xid);
CREATE INDEX idx_xid ON public.branch_table (xid);
Under the same pattern, there cannot be the same index name,sql inside now there is the same index name idx_xid.

Comment (funky-eyes) [reply script/server/db/postgresql.sql:1]: Is your actual business really experiencing indexing conflicts?

Comment (gao1143632516) [reply script/server/db/postgresql.sql:1]: Yes, we resolved the conflict by concatenating the table names in the index names to ensure that the index names are unique

Comment (funky-eyes) [reply script/server/db/postgresql.sql:1]: > Yes, we resolved the conflict by concatenating the table names in the index names to ensure that the index names are unique

Can you tell me exactly which index is conflicting? If it's all by table name combined with column name as you said, why are they conflicting?

Comment (gao1143632516) [reply script/server/db/postgresql.sql:1]: The postgresql.sql file contains two identical index names idx_xid, which cause conflicts 
`
CREATE INDEX idx_xid ON public.lock_table (xid);
CREATE INDEX idx_xid ON public.branch_table (xid);
`

Comment (funky-eyes) [inline script/server/db/postgresql.sql:1]: Please add the pr and author information to the 2.x.md files for both languages in the changes folder.

**Review by slievrly — APPROVED** [review]
LGTM

Pls add change:
https://github.com/apache/incubator-seata/blob/2.x/changes/zh-cn/2.x.md
https://github.com/apache/incubator-seata/blob/2.x/changes/en-us/2.x.md

**Comment by leizhiyuan** [comment]
@gao1143632516 need to solve conflict

**Comment by slievrly** [comment]
@gao1143632516 thanks for you contributions.

**Pull request merged**